src/android/notification/Options.java

```diff
@@ -212,7 +232,20 @@ long getTimeout() {
      * The channel id of that notification.
      */
     String getChannel() {
-        return options.optString("channel", Manager.CHANNEL_ID);
+        // If channel is passed in or we have a low enough SDK for it not to matter, short-circuit.
+        if (!options.optString("channel").isEmpty() || SDK_INT < O) {
```

Comment:
Does this change behaviour? If you currently do not pass in a channel on Android >= O, you will get the default channel with IMPORTANCE_DEFAULT. This seems to change it to the silent channel with IMPORTANCE_LOW?

Reply:
I haven't checked this particular version, because this PR is somewhat old for my codebase, but I did check my latest PR:

https://github.com/katzer/cordova-plugin-local-notifications/pull/1891

... and if you pass no options (no sound, vibrate, etc.), the case statement of the buildChannelWithOptions() gives you a default importance (which means it plays a sound but does not vibrate).